packages/angular-cli/models/webpack-configs/production.ts

```diff
@@ -13,7 +13,6 @@ export const getProdConfig = function (wco: WebpackConfigOptions) {
       new webpack.DefinePlugin({
         'process.env.NODE_ENV': JSON.stringify('production')
       }),
-      new webpack.LoaderOptionsPlugin({ minimize: true }),
```

Comment:
Of the loaders currently used only `css-loader`, `less-loader`, and `sass-loader` use the `minimize` setting.  The last two use the setting to turn on internal minification, which isn't need if `cssnano` is used (or `css-loader` with `minimize` enabled).
`less.js` also deprecated the underlying `compress` option that the `less-loader` is using via the `minimize` setting.